### PR TITLE
Update SubWallet guide screnshots

### PR DESCRIPTION
**Pull request by EmilFattakhov**
Resolves [the issue 763](https://github.com/autonomys/subspace-docs/issues/763)

### CODE REVIEW

File: docs/farming-&-staking/wallets/subwallet.md [hunk 1]
```diff
@@ -35,36 +35,52 @@ If you would like to Import an **Existing** Wallet, then select `Import an accou
...
+1. Open SubWallet, scroll all the way down and click on *Manage tokens*.
 
-    ![wallet-4](/img/doc-imgs/subwallet/Subwallet-8.png)
+![wallet-1](/img/doc-imgs/subwallet/Subwallet-8.png)
+
+2. You can deselect all automatically enabled tokens for your convenience.
+
+![wallet-2](/img/doc-imgs/subwallet/Subwallet-9.png)
+
+3. In a search bar, start typing the name of the network, e.g. Autonomys.
```
Comment (jim-counter) [inline docs/farming-&-staking/wallets/subwallet.md:72]:
```suggestion
3. In the search bar, start typing the name of the network, e.g. Autonomys.
```

File: docs/farming-&-staking/wallets/subwallet.md [hunk 2]
```diff
@@ -35,36 +35,52 @@ If you would like to Import an **Existing** Wallet, then select `Import an accou
...
+![step-6](/img/doc-imgs/subwallet/Subwallet-4.png)
 
-## Connect wallet to Subspace Gemini 3h and find wallet public address
+7. Enter a name for your account.
 
-1. Open SubWallet, click on *Manage tokens*.
+![step-7](/img/doc-imgs/subwallet/Subwallet-5.png)
 
-    ![wallet-1](/img/doc-imgs/subwallet/Subwallet-5.png)
+8. Write down your seed phrase in a secure place. You will need it if you ever forget a password, or decide to transfer the account to another PC.
```
Comment (jim-counter) [inline docs/farming-&-staking/wallets/subwallet.md:54]:
```suggestion
8. Write down your seed phrase in a secure place. You will need it if you ever forget your password, or decide to transfer the account to another PC. 
```

File: docs/farming-&-staking/wallets/subwallet.md [hunk 3]
```diff
@@ -35,36 +35,52 @@ If you would like to Import an **Existing** Wallet, then select `Import an accou
...
+![step-7](/img/doc-imgs/subwallet/Subwallet-5.png)
 
-    ![wallet-1](/img/doc-imgs/subwallet/Subwallet-5.png)
+8. Write down your seed phrase in a secure place. You will need it if you ever forget a password, or decide to transfer the account to another PC. 
 
-2. In a search bar, start typing the name of the network, e.g. Subspace Gemini 3h.
+![step-8](/img/doc-imgs/subwallet/Subwallet-6.png)
 
-    ![wallet-2](/img/doc-imgs/subwallet/Subwallet-6.png)
+9. Congratulations you have fully created your wallet with SubWallet.
```
Comment (jim-counter) [inline docs/farming-&-staking/wallets/subwallet.md:58]:
```suggestion
9. Congratulations you have successfully created your wallet with SubWallet.
```

File: docs/farming-&-staking/wallets/subwallet.md [hunk 4]
```diff
@@ -35,36 +35,52 @@ If you would like to Import an **Existing** Wallet, then select `Import an accou
...
+3. In a search bar, start typing the name of the network, e.g. Autonomys.
+You will see two networks - **Mainnet** and **Taurus** test network.
+
+![wallet-3](/img/doc-imgs/subwallet/Subwallet-10.png)
+
+4. Return to the first screen, where you will see the AI3 token along with your balance. To view your wallet's public address, click on its name.
+
+![wallet-4](/img/doc-imgs/subwallet/Subwallet-11.png)
+
+5. And then click on *Copy* sign to copy the public address.
```
Comment (jim-counter) [inline docs/farming-&-staking/wallets/subwallet.md:81]:
```suggestion
5. And then click on the *Copy* icon to copy the public address. 
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Content looks great. A few suggested typo fixes.

There is a section:

> In this example we will be using https://nova-0.gemini-3h.subspace.network/ws

The screenshot does not match the RPC - would be great if we could fix that too.

**Comment by EmilFattakhov** [comment]
Thanks @jim-counter, the RPC guide has also been updated 👍

**Review by jim-counter — APPROVED** [review]
Thanks Emil!

**Pull request merged**